apps/src/templates/codeDocs/Example.jsx

```diff
@@ -36,17 +36,24 @@ export default function Example({example, programmingEnvironmentName}) {
       const embedUrl = example.app.endsWith('embed_app_and_code')
         ? example.app
         : example.app + '/embed_app_and_code';
+      const enteredHeight =
+        Number(example.embed_app_with_code_height) > 310
+          ? Number(example.embed_app_with_code_height)
+          : Number(example.embed_app_with_code_height) * 1.4;
       return (
         <div style={{width: '100%'}}>
           <div>
             {content}
-            <iframe
-              src={embedUrl}
-              style={{
-                width: '100%',
-                height: Number(example.embed_app_with_code_height) || 310
-              }}
-            />
+            <div style={{height: enteredHeight, overflow: 'scroll'}}>
+              <iframe
+                src={embedUrl}
+                style={{
+                  width: '100%',
+                  height: enteredHeight * 1.5,
```

Comment:
This multiplier means that the user will be able to scroll through 50% more of the code.

Reply:
Does this mean there is a 'cap' on how much we can scroll (assuming minimum height (400), we could scroll a max of 600px)?

Reply:
I think this is a good solution - just wanted to make sure I fully understood it!